File: backjoon/2869.cpp
#include <iostream>
using namespace std;

int main()
{
	int A, B, V;
	cin >> A >> B >> V;
	int day;

	if ((V - A) % (A - B) == 0)
		day = (V - A) / (A - B);
	else
		day = (V - A) / (A - B) + 1;

	cout << day + 1;
}

File: backjoon/2869.py
import sys

input = sys.stdin.readline 

A, B, C = map(int, input().split())

day = 1

# 도착지점과 동일하게 등반 
if (C - A) % (A - B) == 0: 
    day += (C - A) // (A - B)
# 도착지점 추월
else:
    day += (C - A) // (A - B) + 1

print(day)
    